Bookish/Controllers/BookController.cs

```diff
@@ -152,8 +177,45 @@ public async Task<ActionResult> BookInput(BookInput input)
         }
         return View();
     }
+
+    [HttpGet]
+    public async Task<ActionResult> BookEdited(Book input)
```

Comment:
I would rename this method to describe what it does, eg `EditBook`. Additionally, you're making changes to the database, so it should be an `[HttpPost]`, not GET